client/src/components/QandA/QuestionsList/QuestionCard/QuestionDetails.jsx

```diff
@@ -8,7 +8,7 @@ const QuestionDetails = ({ asker, date, helpful, reported, question_id, product_
   const newDate = new Date(date).toLocaleDateString('default', options);
 
   return (
-    <section>
+    <section className='QA-question-details'>
```

Comment:
good use of <section> tag!

Reply:
Thanks, I still don't fully comprehend it but didn't wanna only use DIVs